services/iam/Gemfile

```diff
@@ -22,5 +22,5 @@ end
 group :development, :test do
   gem 'brakeman', require: false
   gem 'spring-commands-rspec'
-  gem 'database_cleaner'
```

Comment:
If we are still loading the configuration for DatabaseCleaner, I don't think this should go away, right?